portal/static/js/profile.js

```diff
@@ -388,10 +388,10 @@
                         var m = "", d = "", y = "", displayDeceasedDate = "";
                         if (data.deceasedDateTime) {
                             var deceasedDateObj = new Date(data.deceasedDateTime);
-                            m = pad(deceasedDateObj.getUTCMonth()+1);
+                            m = self.pad(deceasedDateObj.getUTCMonth()+1);
                             d = deceasedDateObj.getUTCDate();
                             y = deceasedDateObj.getUTCFullYear(); /*global pad*/
-                            displayDeceasedDate = self.modules.tnthDates.displayDateString(pad(m), pad(d), y);
+                            displayDeceasedDate = self.modules.tnthDates.displayDateString(self.pad(m), self.pad(d), y);
```

Comment:
If you haven't already, it would probably be cleaner to standardize on passing around datetime objects (than strings)...